src/app/batch_txn_tool/batch_txn_tool.ml

```diff
@@ -390,23 +387,32 @@ let output_there_and_back_cmds =
             transactions, if this is not present then we use the env var \
             MINA_PRIVKEY_PASS"
          (optional string)
+     and config_file = Cli_lib.Flag.config_files
```

Comment:
Clarification: commit f0704751ebb (_Remove default_transaction_fee from node_config_) also adds `--config-file` to the `batch_txn_tool` app